docs/modules/deploy/pages/versioning-compatibility.adoc

```diff
@@ -34,13 +34,13 @@ However, note the following exceptions:
 Hazelcast Platform has been tested on the following system architectures.
 
 |===
-|Architecture | Platform {full-version}
+|Architecture | Supported
 
 |x86_64
-|✓
+|From Platform 5.3 and later
```

Comment:
I'd suggest:

From Platform 5.3

OR

Version 5.3 and later

Reply:
Thanks, I initially had **From Version 5.3** and will revert to this